lib/resty/core/regex.lua

```diff
@@ -154,6 +156,24 @@ local function get_max_regex_cache_size()
 end
 
 
+function ngx.re.opt(option, value)
+    if option == "jit_stack_size" then
+        local result = C.ngx_http_lua_set_jit_stack_size(value)
+
```

Comment:
I think we should test good return value here first, since it's the most common case. Also, please use the `rc` variable name here instead of `result`.